### PR TITLE
Combine aliens and shorten methods

### PR DESCRIPTION
**Pull request by harveyvanveltom**

### CODE REVIEW

**Comment by Mari3** [comment]
Why is the aliens vector passed as a parameter for the createAliens method? I think that's not neccesary because that vector is an attribute of the game class

**Comment by harveyvanveltom** [comment]
you were right. I now removed the parameter from the createAliens method.

File: src/main/java/spaceinvaders/spaceinvaders_framework/AlienType2.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+     * and returns it. overrides the standard method because this bullet
+     * moves faster and has a different sprite.
+     *
+     * @return Bullet newBullet
+     */
+    @Override
+    public Bullet shoot() {
+      final SpriteSheet spritesheet = new SpriteSheet(getGame().getSpriteSheet());
+      final Bullet newBullet = new Bullet(getX() + 5, getY() + 2, spritesheet);
+      Game.logfile.writeShoot("Alien", getX(), getY());
```
Comment (JerPk) [inline src/main/java/spaceinvaders/spaceinvaders_framework/AlienType2.java]: Maybe you can add the type of alien to the logfile.

File: src/main/java/spaceinvaders/spaceinvaders_framework/AlienType3.java [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+     * and returns it. overrides the standard method because this bullet
+     * moves faster and has a different sprite.
+     *
+     * @return Bullet newBullet
+     */
+    @Override
+    public Bullet shoot() {
+      final SpriteSheet spritesheet = new SpriteSheet(getGame().getSpriteSheet());
+      final Bullet newBullet = new Bullet(getX() + 5, getY() + 2, spritesheet);
+      Game.logfile.writeShoot("Alien", getX(), getY());
```
Comment (JerPk) [inline src/main/java/spaceinvaders/spaceinvaders_framework/AlienType3.java]: Same as in Alientype2.java (add type of alien)

**Comment by JerPk** [comment]
Can be merged after the changes for the logfile

**Pull request merged**